app/src/pages/inside/launchesPage/launchesPage.jsx

```diff
@@ -653,6 +653,7 @@ export class LaunchesPage extends Component {
                   onResetFilter={onResetFilter}
                   onChangeSorting={this.handleChangeSorting}
                   sortingString={sortingString}
+                  userId={userId}
```

Comment:
Looks like it's not used in `LaunchFiltersToolbar`.

Reply:
let me check